src/layouts/ResourceRoom/ResourceRoom.tsx

```diff
@@ -187,44 +207,137 @@ const ResourceRoomContent = ({
-          ) : (
-            directoryData &&
-            directoryData.length > 0 && (
-              <SimpleGrid columns={3} spacing="1.5rem">
-                {directoryData.map(({ name }) => (
-                  <CategoryCard title={name} />
-                ))}
-              </SimpleGrid>
-            )
-          )}
-        </Skeleton>
-      </Section>
-    </SiteViewLayout>
+    <>
+      <SiteViewLayout>
+        <Section>
+          <Box>
+            <HStack spacing="0.25rem" alignItems="center">
+              <Text as="h2" textStyle="h2">
+                Resources
+              </Text>
+              <ContextMenu>
+                <ContextMenu.Button position="relative" bottom={0} right={0} />
+                <ContextMenu.List>
+                  <ContextMenu.Item
+                    icon={<BiWrench fill="icon.alt" />}
+                    onClick={onOpen}
+                  >
+                    <Text mr="3.5rem" color="text.body">
+                      Room settings
+                    </Text>
+                  </ContextMenu.Item>
+                </ContextMenu.List>
+              </ContextMenu>
+            </HStack>
+            <ResourceBreadcrumb />
+          </Box>
+        </Section>
+        <Section>
+          <Box w="full">
+            <SectionHeader label="Resource Categories">
+              <CreateButton as={RouterLink} to={`${url}/createDirectory`}>
+                Create category
+              </CreateButton>
+            </SectionHeader>
+            <SectionCaption icon={BiBulb} label="PRO TIP: ">
+              Categories impact navigation on your site. Organise your resources
+              by creating categories.
+            </SectionCaption>
+          </Box>
+          <Skeleton isLoaded={isLoading} w="100%">
+            {directoryData.length === 0 ? (
+              <Text>No Categories.</Text>
+            ) : (
+              directoryData &&
+              directoryData.length > 0 && (
+                <SimpleGrid columns={3} spacing="1.5rem">
+                  {directoryData.map(({ name }) => (
+                    <CategoryCard title={name} />
+                  ))}
+                </SimpleGrid>
+              )
+            )}
+          </Skeleton>
+        </Section>
+      </SiteViewLayout>
+
+      <Modal isOpen={isOpen} onClose={onClose}>
+        <ModalOverlay />
+        <ModalContent>
+          <form
+            onSubmit={handleSubmit(async (data) => {
+              await updateResourceRoomName(data)
+              onClose()
+            })}
+          >
+            <ModalHeader>
+              <Text>Edit resource room settings</Text>
+            </ModalHeader>
+            <ModalCloseButton />
+            <ModalBody>
+              <FormControl isRequired>
+                <FormLabel>Resource room title</FormLabel>
+                <Input
+                  placeholder="New resource room name"
+                  {...register("newDirectoryName", { required: true })}
```

Comment:
Out of curiosity ya, if we as a team agree that we are ok with prop spreading, should we just change this in our `.eslintrc`, feels neater than having multiple comments to ignore this particular rule, wdyt?

Reply:
hmm so actually the way this panned out was that i was the one that introduced ts and i set a stricter linting than was necessary (so that we get used to having higher standards) and my thinking at that time was that it was easier to go from strict -> loose rather than the other way around (as you won't know what additional checks are useful w/o stumbling upon them). 

so because of that, i found that the prop spreading is a false positive **when using rhf** as the docs advocated the prop spreading way and hence, changed it from error -> warning. the multiple comments to ignore is from when it was an **error** and it **shouldn't** be required now. the reason i didn't remove them was that i felt they were not very confusing + it's drudgery that doesn't really add value but you are correct in that we should remove them...

if you feel that they should be removed, we could file an issue and i could tackle it! wdyt?